src/lib/rpc/svc_auth_gss.c

```diff
@@ -297,7 +297,7 @@ svcauth_gss_validate(struct svc_req *rqst, struct svc_rpc_gss_data *gd, struct r
 	struct opaque_auth	*oa;
 	gss_buffer_desc		 rpcbuf, checksum;
 	OM_uint32		 maj_stat, min_stat, qop_state;
-	u_char			 rpchdr[128];
```

Comment:
Up to 432 bytes might be written in `rpchdr`, but the array is only 128 bytes long.